spec/adapters/googlecharts_spec.rb

```diff
@@ -30,6 +30,14 @@
       ['Lg. Magellanic Cloud', 50000, 120.9],
       ['Bootes I', 60000, 1223.1]
     ]
+    @query_string = 'SELECT A, H, O, Q, R, U LIMIT 5 OFFSET 8'
```

Comment:
Please write  examples in `dummy_iruby` and share the link ( [like](http://nbviewer.jupyter.org/github/sciruby/daru-view/tree/master/spec/dummy_iruby/)).

Reply:
I  am expecting notebook link like [this](http://nbviewer.jupyter.org/github/Prakriti-nith/daru-view/blob/import_spreadsheet/spec/dummy_iruby/Google%20Chart%20%7C%20Data%20from%20google%20spreadsheet.ipynb) .

Reply:
Here is the [link](http://nbviewer.jupyter.org/github/Prakriti-nith/daru-view/blob/d2bb96be63aaeac2f79cebbac748162c89f85daa/spec/dummy_iruby/Google%20Chart%20|%20Data%20from%20google%20spreadsheet.ipynb) to view the examples.
I am getting the outputs for all the jupyter notebook cells locally but I can't understand why they are not visible online.

Reply:
Did you try in incognito mode ?

Reply:
Yes, I tried the examples in incognito mode. Running each cell individually is showing the correct output but after I save the file and open it again, then only the last output is visible.